pkgs/by-name/ch/chart-releaser/package.nix

```
@@ -0,0 +1,67 @@
+{ buildGoModule
+, coreutils
+, fetchFromGitHub
+, git
+, installShellFiles
+, kubernetes-helm
+, lib
+, makeWrapper
+}:
+
+buildGoModule rec {
+  pname = "chart-releaser";
+  version = "1.6.1";
+
+  # 0102fa30dd78df0d0f8093c4607e6080972ff82d
+
+  src = fetchFromGitHub {
+    owner = "helm";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-8+O9JErEB1Z/zlrWm975v5Qf0YG0lbPcjY5LlDKw8U4=";
+
+  };
+
+  # Don't run tests.
+  doCheck = false;
+  doInstallCheck = false;
+
+  vendorHash = "sha256-S/V1kTgD/cXaJNYpPPNjM9ya2zv6Bsy/YBn7I/1EjwI=";
+
+  postPatch = ''
+    substituteInPlace pkg/config/config.go \
+      --replace "\"/etc/ct\"," "\"$out/etc/ct\","
```

> substituteStream(): WARNING: '--replace' is deprecated, use --replace-{fail,warn,quiet}. (file 'pkg/config/config.go')
substituteStream(): WARNING: pattern \"/etc/ct\"\, doesn't match anything in file 'pkg/config/config.go'

```suggestion
    substituteInPlace pkg/config/config.go \
      --replace-fail "/etc/cr" "$out/etc/cr"
```